tasks/generate.yml

```diff
@@ -45,3 +45,4 @@
       tags:
         - ssh-keys-generate-public-keys
   connection: local
+  become: "{{ ssh_keys_generate_local_become }}"
```

Comment:
What do you think about?

become: "{{ ssh_keys_generate_keys_local_become | default(omit) }}"